### ISSUE TITLE
Unable to load the Pretrained Models

### ISSUE THREAD

**Issue by abreorussel** [jwgdmkj/LMLT#2]
When I am trying the to load the pretrained models , it gives an error stating that there are missing keys in the state dictionary.
![image](https://github.com/user-attachments/assets/ff59ef87-c767-43c7-a510-bf143a96a2b2)

**Comment by jwgdmkj** [comment]
Sorry, I uploaded the file old. I modified it. If you have any problems, please let me know again.

**Comment by abreorussel** [comment]
Thank you . It works now . If you dont mind could you also add the pretrained models for the other architectures i.e tiny , base and the large version (x2, x3 , x4) . I suppose you have added for tiny_x2 alone .

**Comment by jwgdmkj** [comment]
Sorry. I uploaded the wrong file. Thank you for letting me know. I uploaded a new file. If you have any problems, please let me know again. Thank you.

**Comment by abreorussel** [comment]
I am able to load the rest of the models as well. Thank you .

**Comment by Carry-w** [comment]
@abreorussel  Brother, can you run the test set through its pre training model? I have been unable to run. I don't know what the structure of his test set is and where the test set is located. If you can run, can you show me. Thank you, brother.

**Comment by abreorussel** [comment]
@Carry-w You will find the respective yaml files in options/test/LMLT. Based on the pretrained model you want to use , you have to configure the file as per your need. In the yml file , against pretrain_network_g , specify the path to the corresponding pretrained model pth file. Make sure the path to the datasets are alright. Once that is done just run python3 basicsr/test.py -opt options/test/LMLT/test_tiny(base, large)_benchmark_X2(3, 4).yml based on the respective model/yaml file under consideration.
